Source/FieldSolver/FiniteDifferenceSolver/HybridPICSolveE.cpp

```diff
@@ -586,8 +586,9 @@ void FiniteDifferenceSolver::HybridPICSolveECylindrical (
 
                 // Interpolate current to appropriate staggering to match E field
                 Real jtot_val = 0._rt;
+                Real jr_val = 0._rt;
                 if (solve_for_Faraday && resistivity_has_J_dependence) {
-                    const Real jr_val = Interp(Jr, Jr_stag, Er_stag, coarsen, i, j, 0, 0);
+                    jr_val = Interp(Jr, Jr_stag, Er_stag, coarsen, i, j, 0, 0);
```

Comment:
This will only be evaluated if `resistivity_has_J_dependence == true`, but it needs to be evaluated `if (solve_for_Faraday && include_hyper_resistivity_term)`, and the same for the other components.

Reply:
Thanks for catching this.